### PR TITLE
Style sidebar

### PR DESCRIPTION
**Pull request by henryzarza**
## Summary

- Correction in `Sidebar` style.
- Fix in padding of the general container.

## Screenshots

sidebar

### CODE REVIEW

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -27,61 +34,124 @@ export default {
...
+  transition: all $transition-duration $transition-function;
+  width: 100%;
+
+  &:hover {
+    background-color: $white;
+    color: $science-blue;
+  }
 }
+
+.icon-palette {
```
Comment (nicolobruno) [inline src/components/Sidebar.vue]: You can use `m-right-1` instead of creating the class

File: src/components/Feed.vue [hunk 2]
```diff
@@ -35,14 +35,15 @@ export default {
 @import 'components/buttons/btn-5';
 
 .feed-content {
-  width: 100%;
-  margin: 60px auto;
+  margin: 0 auto;
   max-width: 1080px;
+  padding: 80px 20px 20px 80px;
   width: 100%;
```
Comment (tomiir) [inline src/components/Feed.vue]: yo can use `full-width` here

File: src/components/Feed.vue [hunk 3]
```diff
@@ -35,14 +35,15 @@ export default {
...
 }
 
 .components-feed {
   display: flex;
```
Comment (tomiir) [inline src/components/Feed.vue]: If you use a generic class `row` then all of this is set automatically. Also, wrap is the default value.

File: src/components/Sidebar.vue [hunk 4]
```diff
@@ -2,22 +2,29 @@
   .sidebar-container
     .sidebar-upper-section
       .sidebar-header
-        h2.sidebar-title.text-big
-          | Frontend Cookbook
+        h2.sidebar-title.sidebar-title-wolox
+          | wolox
```
Comment (tomiir) [inline src/components/Sidebar.vue]: Is lowercase OK?

File: src/components/Sidebar.vue [hunk 5]
```diff
@@ -27,61 +34,124 @@ export default {
   },
   methods: {
     showNewList(list) {
-      this.$emit('list', list.component);
+      this.$emit("list", list.component);
```
Comment (tomiir) [inline src/components/Sidebar.vue]: don't really know what this does, but shouldn't this be a constant of some sort?

File: src/components/Sidebar.vue [hunk 6]
```diff
@@ -27,61 +34,124 @@ export default {
...
   max-width: 300px;
-  padding: 90px 20px 10px;
   position: sticky;
   top: 0;
   width: 100%;
 }
 
 .sidebar-header {
-  height: 180px;
+  display: flex;
```
Comment (tomiir) [inline src/components/Sidebar.vue]: use `column` generic class

File: src/components/Sidebar.vue [hunk 7]
```diff
@@ -27,61 +34,124 @@ export default {
...
+  transition: all $transition-duration $transition-function;
 
   &:not(:last-child) {
     margin-bottom: 20px;
   }
+
+  &::before {
+    background-color: $white;
+    content: '';
+    height: 100%;
```
Comment (tomiir) [inline src/components/Sidebar.vue]: do we have a `full-height` class?

File: src/components/Sidebar.vue [hunk 8]
```diff
@@ -27,61 +34,124 @@ export default {
...
+  text-transform: uppercase;
+  width: auto;
+}
+
+.sidebar-footer-set-style {
+  align-items: center;
+  background-color: $science-blue;
+  color: $white;
+  cursor: pointer;
+  display: flex;
```
Comment (tomiir) [inline src/components/Sidebar.vue]: `row`

File: src/components/Sidebar.vue (hunk 4 above)
Comment (henryzarza) [reply src/components/Sidebar.vue]: Yes, because with CSS i transform it to uppercase

File: src/components/Sidebar.vue (hunk 7 above)
Comment (henryzarza) [reply src/components/Sidebar.vue]: It isn't possible because is a `::before`

File: src/components/Feed.vue [hunk 9]
```diff
@@ -1,21 +1,21 @@
...
     h2.title.m-bottom-4
       | {{ title }}
-    .components-feed
+    .row.wrap
       card(v-for='(comp, index) in componentsToShow' :component='comp' :number='index' :key='comp.title')
 </template>
 
 <script>
-import Card from './Card'
+import Card from "./Card";
```
Comment (mcallegari10) [inline src/components/Feed.vue]: No need for the semicolon and leave the single quotes

File: src/components/Feed.vue [hunk 10]
```diff
@@ -1,21 +1,21 @@
...
 
 export default {
   props: {
     title: { type: String, required: true },
     componentsToShow: { type: Array, required: true }
   },
   components: { Card }
-}
+};
```
Comment (mcallegari10) [inline src/components/Feed.vue]: No need for the semicolon 😄

**Pull request merged**